doc/tutorial_ti_cc3200.md

```diff
@@ -279,8 +280,9 @@ Reaching this point means you are able to produce and execute CC3200 compatible
 
 ### Adding the Xively Client to ent_wlan
 
-- locate the successful wifi connection point in the main.c of the ent_wlan example (arond line 647, comment: "//wait for few moments")
-- here put a call on the  ConnectToXively(); function. Its implementation is based on the examples in the Client repo, e.g. xively-client-c/examples/mqtt_logic_producer/src/mqtt_logic_producer.c:
+Next we're going to add a function to connect to the Xively Broker. Its implementation is based on the examples in the Client repo, e.g. `xively-client-c/examples/mqtt_logic_producer/src/mqtt_logic_producer.c`.
+
+- Paste the following code within `main.c` of the _ent_wlan_ anywhere in the main portion of the file
```

Comment:
Does the function definition placement matter if the function call is not part of the main stack and happens on the connection-successful event? Or is this more of a best practices code style guideline?